repos/system_upgrade/common/actors/commonleappdracutmodules/files/dracut/90sys-upgrade/leapp_debug_tools.sh

```diff
@@ -0,0 +1,34 @@
+# library containing some useful functions for debugging in initramfs
+
+# mounts the sysroot
+leapp_dbg_mount() {
+    systemctl start sysroot.mount
+}
+
+# source programs from $NEWROOT, mount if not mounted
+leapp_dbg_source() {
+    systemctl is-active sysroot.mount --quiet || {
+        echo "sysroot not mounted, mounting...";
+        leapp_dbg_mount || return 1
+    }
+    for dir in /usr/local/bin /usr/local/sbin /usr/bin /usr/sbin; do
+        export PATH="$PATH:${NEWROOT}$dir"
```

Comment:
/sysroot/bin and /sysroot/sbin is enough, you won't need /usr/{s,}bin nor /usr/local/{s,}bin

But you probably also want: `export LD_LIBRARY_PATH=/sysroot/lib64`

Reply:
Yeah that's why the shared libs don't work